tests/platform_win32_ut/platform_win32_ut.c

```diff
@@ -215,6 +221,26 @@ TEST_FUNCTION(platform_init_WSAStartup_0_fail)
     // cleanup
 }
 
+#ifndef DONT_USE_UPLOADTOBLOB
```

Comment:
is there a test build which has this enabled?

Reply:
I don't know what are the configurations that we are testing on DevOps.

---
In reply to: [319290351](https://github.com/Azure/azure-c-shared-utility/pull/366#discussion_r319290351) [](ancestors = 319290351)